blogs/youtube.com.c.ggnot2.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "Melissa Van Bussel (ggnot2)'s YouTube channel about R",
+  "type": "blog",
```

Comment:
Can we change this one to `YouTube` (instead of `blog`)?

Reply:
Oops, didn't realize that was an option! Updated :) Thank you!